package.json

```diff
@@ -95,20 +95,24 @@
   ],
   "readmeFilename": "README.md",
   "devDependencies": {
+    "@bunchtogether/vite-plugin-flow": "^1.0.2",
     "@commitlint/cli": "^16.2.1",
     "@commitlint/config-conventional": "^16.2.1",
     "@krakenjs/grumbler-scripts": "^8.0.4",
-    "@krakenjs/sync-browser-mocks": "^3.0.0",
-    "@storybook/expect": "^27.5.2-0",
+    "@vitest/coverage-c8": "^0.29.2",
+    "@vitest/ui": "^0.29.2",
     "cross-env": "^7.0.3",
     "flow-bin": "0.135.0",
     "flow-typed": "^3.8.0",
     "husky": "^7.0.4",
     "jest": "^29.3.1",
+    "jsdom": "^21.1.1",
     "lint-staged": "^12.4.0",
-    "mocha": "^4.0.0",
+    "msw": "^1.1.0",
```

Comment:
Is Vitest using this?  I don't see it imported anywhere in the code.

Reply:
good catch, i tried it for the mocks but ended up not using it. Will remove now